src/org.opensuse.yast.GPMC.desktop

```diff
@@ -16,7 +16,7 @@ X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=gpmc
 
 Icon=yast-gpmc
-Exec=yast2 '@CLIENTDIR@/gpmc.py'
+Exec=/usr/bin/xdg-su -c 'yast2 gpmc qt'
```

Comment:
why qt only? usually we have this line without 'qt' specified.

Reply:
Fixed. I think I copy/pasted that from somewhere.